### PR TITLE
Add .eslintrc files for modules 2 and 3

### PR DESCRIPTION
**Pull request by damwhit**
* Moving .eslintrc files to this style guide so that it can serve as single source of truth for students
  * Updated readme with cohort specific links
  * Mod 2 eslint file lived here: https://github.com/turingschool-examples/react-starter-kit/blob/master/.eslintrc
  * Mod 3 eslint file lived here: https://gist.github.com/brittanystoroz/860f521de95c27b8d07aff8f2f27485e

### CODE REVIEW

File: linters/module-2/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  "plugins": [
+    "react"
+  ],
+  "env": {
+    "browser": true,
+    "mocha": true,
+    "node": true,
+    "es6": true
+  },
+  // Having a problem with one of these rules? Learn more about it here: https://eslint.org/docs/rules/
```
Comment (letakeane) [inline linters/module-2/.eslintrc]: Sorry, this needs to be removed from ours. Will you kill this line?

Comment (damwhit) [reply linters/module-2/.eslintrc]: No need to be sorry! Done!

**Comment by damwhit** [comment]
@letakeane, @joshuajhun, and @brittanystoroz please let me know if this is gtm! The intention is to let this repo serve as the single source of truth for syntax, style, and linting.

File: README.md [hunk 2]
```diff
@@ -7,10 +7,20 @@ other style guides
...
 
-### FEE Module 1
+### Cohort Specific
+
+#### FEE Module 1
 
   - [JavaScript](es5/)
 
+#### FEE Module 2
+
```
Comment (nfosterky) [inline README.md:17]: Non-React Projects: https://github.com/turingschool-examples/game-time-starter-kit-FEm1/blob/master/.eslintrc.json

**Pull request merged**